src/compute/src/compute_state.rs

```diff
@@ -162,6 +163,9 @@ pub struct ComputeState {
     /// Interval at which to perform server maintenance tasks. Set to a zero interval to
     /// perform maintenance with every `step_or_park` invocation.
     pub server_maintenance_interval: Duration,
+
+    /// The time at which to expire replicas.
```

Comment:
This is probably a good place to describe a bit what "expiring a replica" means.